me.hanh.zwallet.Ywallet.yml

```diff
@@ -0,0 +1,48 @@
+id: me.hanh.zwallet.Ywallet
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: ywallet
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-data/databases:create
+modules:
+  - name: zwallet_build
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/hhanh00/zwallet/releases/download/v1.2.15/zwallet.tgz
+        sha256: 4b6cbf8056cc2dbbdebd70d5730fb864b12074a9f6d184adf7d2b1939327a9a0
+        strip-components: 0
+    build-commands:
+      - mkdir /app/bin
+      - cp -r * /app/bin
+  - name: warp_api_ffi
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/hhanh00/zwallet/releases/download/v1.2.15/libwarp_api_ffi.so
```

Comment:
Given that both aarch64 and x86_64 flatpak are built I expect this to be a problem.

Reply:
Added only_arches

Reply:
it doesn't do what you think it does. it just skips.